pkg/controller/vsphere/reconciler_test.go

```diff
@@ -1400,17 +1400,32 @@ func TestDelete(t *testing.T) {
 				t.Errorf("expected error on the first call to delete")
 			}
 
-			moTask, err := reconciler.session.GetTask(reconciler.Context, reconciler.providerStatus.TaskRef)
+			powerOffTask, err := reconciler.session.GetTask(reconciler.Context, reconciler.providerStatus.TaskRef)
 			if err != nil {
 				if !isRetrieveMONotFound(reconciler.providerStatus.TaskRef, err) {
 					t.Fatal(err)
 				}
 			}
-			if moTask.Info.DescriptionId != "VirtualMachine.destroy" {
-				t.Errorf("task description expected: VirtualMachine.destroy, got: %v", moTask.Info.DescriptionId)
+			// first run should schedule power off
+			if powerOffTask.Info.DescriptionId != "VirtualMachine.powerOff" {
```

Comment:
Is there a const we can use instead of this `VirtualMachine.powerOff` string?

Reply:
I didn't find anything in underlying lib. Can add it there: https://github.com/openshift/machine-api-operator/blob/master/pkg/controller/vsphere/reconciler.go#L51